src/lib/DataConnectStore.js

```diff
@@ -59,6 +59,22 @@ export default class DataConnectStore {
     return useCase.connectors.map(c1 => this.find(c2 => c1.slug === c2.slug))
   }
 
+  // Fetch all accounts and updates their matching connectors
+  fetchAllAccounts (index) {
+    if (!index && this.accountsIndex) index = this.accountsIndex
```

Comment:
Question: would it be possible to have `fetchAllAccounts` forget about the index and have the index logic in `getAllAccounts` (getAllAccounts would only need to check this.accountsIndex instead of taking it in parameter) ? To me the index part is an implementation detail and should be kept hidden.